Cartfile

```diff
@@ -1 +1,6 @@
-github "mapbox/MapboxDirections.swift" "swift3"
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.4
+github "mapbox/MapboxDirections.swift" "master"
+github "mapbox/MapboxGeocoder.swift" "master"
+github "52inc/Pulley" "master"
+github "rs/SDWebImage" "master"
```

Comment:
We should try to pin to a specific version (with the tadpole operator) to avoid surprises when running `carthage update`.